bom/application/pom.xml

```diff
@@ -6443,16 +6443,11 @@
                 <groupId>io.opentelemetry.semconv</groupId>
                 <artifactId>opentelemetry-semconv</artifactId>
                 <version>${opentelemetry-semconv.version}</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>io.opentelemetry</groupId>
-                        <artifactId>opentelemetry-bom</artifactId>
-                    </exclusion>
-                    <exclusion>
-                        <groupId>io.opentelemetry</groupId>
-                        <artifactId>opentelemetry-api</artifactId>
-                    </exclusion>
-                </exclusions>
```

Comment:
With this, you do a bit more than adding something to the BOM as advertised in the title. Could you clarify?

I'm all for what you added below but not sure I would be comfortable merging the part above without having @brunobat checking as it's something that has been added explicitly.

Reply:
opentelemetry-semconv depends on neither opentelemetry-bom nor opentelemetry-api so the exclusions are completely superfluous. Perhaps they were dependencies in the past. The change is in a separate commit.